design/peer-awareness.md

```diff
@@ -0,0 +1,64 @@
+# Peer Awareness
+
+Target version: Yorkie 0.1.2
+
+## Use case
```

Comment:
What about `Overview` or `Goals` instead of `Use case`? I think that this section is more of an explanation of what problems `peer awareness` solves, rather than talking about use cases. What do you think?

Reply:
I think the goals could be suitable as well. But I wanted to write what value it would give the user with Peer Awareness.

Reply:
Agree with @dc7303. Additionally I thought `Motivation` would suit this section as well, since this section mentions what's driving this feature.